telegraf/1.2/alpine/Dockerfile

```diff
@@ -5,7 +5,7 @@ ENV TELEGRAF_VERSION 1.2.1
 RUN apk add --no-cache ca-certificates && \
     update-ca-certificates
 
-RUN apk add --no-cache --virtual .build-deps wget gnupg tar && \
+RUN apk add --no-cache --virtual .build-deps wget gnupg tar iputils && \
```

Comment:
Can you move this step to another step? The `apk del` is important as it uninstalls tools that aren't used after downloading the binary and the gpg key. If you can just move the iputils installation to the same section that installs `ca-certificates` and keep the rest the same, this is probably fine.